processes/managing-reviewers.md

```diff
@@ -0,0 +1,49 @@
+# Managing Reviewers
+
+## Status
+
+implementable
+
+## Summary
+
+After the migration to use OWNERS files to manage reviewers, it is
+easier for us to add reviewers to separate repositories. This document
+describes the process for adding reviewers to a metal3 project repo.
+
+### Goals
+
+1. Describe a process for adding reviewers.
+2. Keep the process light-weight.
+
+### Non-Goals
+
+1. Change the process for adding [maintainers](../maintainers/).
+
+## Proposal
+
+Anyone can propose a patch to update an OWNERS file in a repository to
+add a reviewer. The patch should be submitted as a standalone PR,
+rather than being linked to any other contribution.
+
+The reviewer list for each repository will be pruned over time to
+remove contributors who are inactive.
```

Comment:
should we put a clear criteria (like no commits in the last x months for example) to make it objective ?

Reply:
I always hesitate to make hard rules like that. What happens in a stable repo for a feature-complete component, where there is no real need to change code for several months in a row? We wouldn't want to remove all of the reviewers.

Reply:
ok. Then I guess anybody can propose to prune the reviewer list as long as it is approved by one approver and one reviewer as mentioned by Russell ?

Reply:
That seems OK to me. I would expect anyone reviewing a patch to the OWNERS file to be careful with approving it.